tests/events/listStatusChangesAfter.test.js

```diff
@@ -1,240 +1,188 @@
-})
-
-test('should list status changes after given id with filter', async () => {
-    let chartKey = testUtils.getChartKey()
-    await testUtils.createTestChart(chartKey, user.secretKey)
-    let event = await client.events.create(chartKey)
-    let holdToken = await client.holdTokens.create()
-    await client.events.book(event.key, 'A-1')
-    await client.events.hold(event.key, 'A-2', holdToken.holdToken)
-    await client.events.hold(event.key, 'B-1', holdToken.holdToken)
-    await client.events.book(event.key, 'A-3')
-    let firstPage = await client.events.statusChanges(event.key).firstPage()
-
-    let params = new StatusChangesParams().withFilter('1')
-    let pageAfter = await client.events.statusChanges(event.key).pageAfter(firstPage.items[0].id, params)
-
-    let labels = [
-        pageAfter.items[0].objectLabel,
-        pageAfter.items[1].objectLabel
-    ]
-    expect(labels).toEqual(['B-1', 'A-1'])
-    expect(pageAfter.items.length).toBe(2)
-    expect(pageAfter.nextPageStartsAfter).toBe(null)
-    expect(pageAfter.previousPageEndsBefore).toBe(pageAfter.items[0].id + '')
-})
-
-test('should list status changes after given id with filter and page size', async () => {
-    let chartKey = testUtils.getChartKey()
-    await testUtils.createTestChart(chartKey, user.secretKey)
-    let event = await client.events.create(chartKey)
-    let holdToken = await client.holdTokens.create()
-    await client.events.book(event.key, 'A-1')
-    await client.events.hold(event.key, 'A-2', holdToken.holdToken)
-    await client.events.hold(event.key, 'B-1', holdToken.holdToken)
-    await client.events.book(event.key, 'A-3')
-    let firstPage = await client.events.statusChanges(event.key).firstPage()
-
-    let params = new StatusChangesParams().withFilter('1')
-    let pageAfter = await client.events.statusChanges(event.key).pageAfter(firstPage.items[0].id, params, 1)
-
-    expect(pageAfter.items[0].objectLabel).toEqual('B-1')
-    expect(pageAfter.items.length).toBe(1)
-    expect(pageAfter.nextPageStartsAfter).toBe(pageAfter.items[0].id + '')
-    expect(pageAfter.previousPageEndsBefore).toBe(pageAfter.items[0].id + '')
-})
-
-test('should not list status changes after given id with unmatched filter', async () => {
-    let chartKey = testUtils.getChartKey()
-    await testUtils.createTestChart(chartKey, user.secretKey)
-    let event = await client.events.create(chartKey)
-    await client.events.book(event.key, 'A-2')
-    await client.events.book(event.key, 'B-2')
-    await client.events.book(event.key, 'C-2')
-    let firstPage = await client.events.statusChanges(event.key).firstPage()
-
-    let params = new StatusChangesParams().withFilter('1')
-    let pageAfter = await client.events.statusChanges(event.key).pageAfter(firstPage.items[0].id, params)
-
-    expect(pageAfter.items).toEqual([])
+describe('list status changes after id', () => {
+    beforeAll(async () => {
+        jest.setTimeout(35000)
```

Comment:
Do we need to set a timeout here? Or should beforeAll timeout after the 30s we set in setupTests.js?

Reply:
setupTest contains `beforeEach`, which runs after `beforeAll` so unless I am wrong, without the timeout, this beforeAll would use the default 500ms